google-cloud-pubsub/lib/google/cloud/pubsub.rb

```diff
@@ -306,12 +308,17 @@ module Cloud
     # pubsub = Google::Cloud::Pubsub.new
     #
     # sub = pubsub.subscription "my-topic-sub"
-    # received_message = sub.pull.first
-    # if received_message
+    # subscriber = sub.listen do |received_message|
     #   puts received_message.message.data
+    #
     #   # Mark for redelivery by setting the deadline to now
     #   received_message.delay! 0
```

Comment:
should this use `received_message.nack!` instead?

Reply:
I think `.nack!` should be preferred over `.delay! 0`.

Reply:
Ok, changing it now.